client/src/components/app.jsx

```diff
@@ -72,10 +91,10 @@ class App extends React.Component {
         <PhotoBody>
           <div className="photo-gallery-body">
             <div>
-              <Header Photos={this.state.photos} />
+              <Header Photos={this.state.photos} handleHeaderClick={this.handleHeaderClick} />
             </div>
```

Comment:
It seems like there are a lot of extra divs in your render functions; if these components (Header for example) return divs, there is no reason to wrap them in an extra div.. the extra divs make it a little harder to see what's happening in the code and in the dev tools element panel